docs/markdown/snippets/install_data_follow_symlink_arg.md

```diff
@@ -0,0 +1,5 @@
+## Added follow_symlinks arg to install_data
+
+The [[install_data]] function now has an optional argument `follow_symlinks`
+that allows following symlinks when installing, as this behaviour is subject to
+change.
```

Comment:
This explanation is unclear. It should have the same explanation for what actually happens as below in `install-data.yaml`.